openvisualizer/motehandler/moteprobe/moteprobe.py

```diff
@@ -1,175 +1,34 @@
-# Copyright (c) 2010-2013, Regents of the University of California. 
-# All rights reserved. 
-#  
+# Copyright (c) 2010-2013, Regents of the University of California.
```

Comment:
I don't think any component in OpenVisualizer needs a instance of the parent class MoteProbe, they all use the instances of the child classes SerialMoteProbe, IotLabMoteProbe, ... we can make this an abstract base class

`import abc`